notebooks/week_3/bouton_density.ipynb

```diff
@@ -260,12 +260,12 @@
    "metadata": {},
    "source": [
     "### Exercise #1\n",
-    "Calculate the average bouton density in an interneuron and a neuron in general. Store the answer in _ans_1a_ and _ans_1b_.\n",
+    "Calculate the average bouton density in an interneuron and a neuron (either excitatory or inhibitory). Store the mean value and the standard deviation in a list each for the interneuron (answer _ans_1a_) and the neuron (answer_ans_1b_).\n",
     "\n",
-    "__Hint__: to query GIDs of interneurons use Cell.SYNAPSE_CLASS. When you query all the interneurons, you do not need any filters.\n",
+    "Hint: to query GIDs of interneurons use Cell.SYNAPSE_CLASS. When you query all the interneurons, you do not need any filters.\n",
     "\n",
     "### Exercise #2\n",
-    "Calculate bouton density not in the central column and store the answer in _ans_2a_ and _ans_2b_. Is it lower, bigger or the same of the bouton density calculated in the central column? Why?"
+    "Calculate the average bouton density in a neuron (either excitatory or inhibitory) in column 0 (zero). Similarly to the previous exercise, store mean and standard deviation in a list and assign it to _ans_2a_. Are the values lower, bigger or the same of the bouton densities calculated in the central column? Store 'lower', 'bigger', or 'same' in _ans_2b_. As for ungraded exercise, can you explain why you found or not found a difference?"
```

Comment:
> in column 0
What this column 0 defined? How do they know which target they have to use?

Reply:
Column 0 is a column in the brain part?